metricbeat/module/php_fpm/php_fpm.go

```diff
@@ -0,0 +1,103 @@
+package php_fpm
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+const (
+	defaultScheme = "http"
+	defaultPath   = "/status"
+)
+
+// HostParser is used for parsing the configured php-fpm hosts.
+var HostParser = parse.URLHostParserBuilder{
+	DefaultScheme: defaultScheme,
+	DefaultPath:   defaultPath,
+}.Build()
+
+// StatsClient provides access to php-fpm stats api
+type StatsClient struct {
+	address  string
+	user     string
+	password string
+	http     *http.Client
+}
+
+// NewStatsClient creates a new StatsClient
+func NewStatsClient(m mb.BaseMetricSet, isFullStats bool) *StatsClient {
+	var address string
+	address = m.HostData().SanitizedURI + "?json"
```

Comment:
You can move the query param to the `URLHostParserBuilder` by setting `QueryParams: "json"`.